tests/test_main.py

```diff
@@ -1206,6 +1206,18 @@ def test_sort(dummy: Tensor) -> Tensor:
     return ep.sort(t)
 
 
+@compare_all
+def test_topk_0(dummy: Tensor) -> Tensor:
+    t = ep.arange(dummy, 27).float32().reshape((3, 3, 3))
```

Comment:
I think it would be good to test on something that's not already sorted.

Reply:
I had copied from the tests for `sort` and `argsort` where one is already sorted and the other reverse sorted, so I tested values on sorted and indices on reverse sorted. But I've changed it to something more complicated.